### PR TITLE
Pull tracing test data from separate repo

### PR DESCRIPTION
**Pull request by antekresic**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -43,10 +43,10 @@ jobs:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
       with:
-        lfs: true
+        lfs: false
 
     - name: Checkout LFS objects
```
Comment (JamesGuthrie) [inline .github/workflows/go.yml:48]: This should probably be renamed

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -102,10 +102,10 @@ jobs:
...
```
Comment (JamesGuthrie) [inline .github/workflows/go.yml:107]: Here too

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

**Review by JamesGuthrie — APPROVED** [review]
LGTM. The real test is in the build though 😂

**Review by niksajakovljevic — APPROVED** [review]

**Comment by antekresic** [comment]
bors ping

**Review by Harkishen-Singh — APPROVED** [review]

**Pull request merged**